components/stripe/sources/new-failed-payment/new-failed-payment.mjs

```diff
@@ -0,0 +1,19 @@
+import common from "../common/webhook-base.mjs";
+
+export default {
+  ...common,
+  key: "stripe-new-failed-payment",
+  name: "New Failed Payment",
```

Comment:
Payment Failed

Reply:
I think this one is ok as well -- "New Failed Payment" makes sense.